realm/realm-library/src/syncTestUtils/kotlin/io/realm/mongodb/sync/SyncConfigurationExt.kt

```diff
@@ -16,8 +16,13 @@
 
 package io.realm.mongodb.sync
 
+import io.realm.RealmModel
 import io.realm.internal.OsRealmConfig
 
-fun SyncConfiguration.Builder.test_sessionStopPolicy(policy: OsRealmConfig.SyncSessionStopPolicy): SyncConfiguration.Builder {
+fun SyncConfiguration.Builder.testSchema(firstClass: Class<out RealmModel>, vararg x: Class<out RealmModel> ) : SyncConfiguration.Builder {
```

Comment:
Not 100% sure I understand why adding this is better than just renaming the `schema` method?

Reply:
The `schema` method is package protected, so all tests using this would have to be in the `io.realm.mongodb.sync` packages. As the tests are also relating on other package protected methods from `io.realm` and `io.realm.mongodb` we cannot resolve all package private access by placing it in one of the packages. Thus adding test-only extension methods to workaround this. Alternative would be to subclass, but as almost all tests are in Kotlin now, I just went for the extensions.

Reply:
Ah okay. That sounds fine then 👍

Reply:
We could even remove these _test-only_ internal methods to reduce the number of symbols now that it is so much easier to extend class from outside.